### DISCUSSION TITLE
Ezra Bible in Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by FlavioTheoLima** [ezra-bible-app/ezra-bible-app#689] [General]
Hello friends!!!
I'm having problems installing the Ezra Bible app on Ubuntu 22.04... Broken dependency: libicu66 missing.

**Comment by drbenedictpaul** [comment]
> Hello friends!!! I'm having problems installing the Ezra Bible app on Ubuntu 22.04... Broken dependency: libicu66 missing.

For me too...

**Reply by tobias-klein** [reply]
A newly configured package is needed for Ubuntu 22.04. This has already been addressed from a development perspective (see #684). Since this issue generates quite some interest I'm going to check whether we can generate another patch release on top of 1.6 - including this new package.

**Reply by FlavioTheoLima** [reply]
Okay, my noble friend!
I'm really glad I found this app!

**Reply by tobias-klein** [reply]
[1.6.2](https://github.com/ezra-bible-app/ezra-bible-app/releases/tag/1.6.2) is available now with added support for Ubuntu 22.04 LTS!

**Comment by drbenedictpaul** [comment]
Hi Tobias,

Please go through the error it shows up while installing on Debian 11.

sudo dpkg -i ezra-bible-app_ubuntu2204-1.6.2.deb 
dpkg-deb: error: archive 'ezra-bible-app_ubuntu2204-1.6.2.deb' uses unknown compression for member 'control.tar.zst', giving up
dpkg: error processing archive ezra-bible-app_ubuntu2204-1.6.2.deb (--install):
 dpkg-deb --control subprocess returned error exit status 2
Errors were encountered while processing:
 ezra-bible-app_ubuntu2204-1.6.2.deb


**OS Details**

Distributor ID: Debian
Description:    Debian GNU/Linux 11 (bullseye)
Release:        11
Codename:    bullseye

**Reply by tobias-klein** [reply]
Dear Benedict,

It seems like you are trying to use the Ubuntu package on Debian. That won't work.
There is a separate package for Debian on the release page. I am pasting the link below:

https://github.com/ezra-bible-app/ezra-bible-app/releases/download/1.6.2/ezra-bible-app_debian11-1.6.2.deb

**Comment by drbenedictpaul** [comment]
Dear Tobias,

Thank you very much. Sorry, I missed it.

Regards.

Dr. Benedict Paul C
Senior Lecturer
Dept of Biotechnology
Faculty of Biomedical Sciences & Technology
Sri Ramachandra Institute of Higher Education and Research
Porur, Chennai - 600116.
Tamilnadu, India.
Website: www.drpaul.cc
Mobile/WhatsApp: +91 7010972354
